pkg/operator/status/status_controller.go

```diff
@@ -125,11 +121,11 @@ func (c StatusSyncer) sync() error {
 		if apierrors.IsNotFound(createErr) {
 			// this means that the API isn't present.  We did not fail.  Try again later
 			klog.Infof("ClusterOperator API not created")
-			c.queue.AddRateLimited(workQueueKey)
+			syncCtx.Queue().AddRateLimited(factory.QueueKey("StatusSyncer_" + c.clusterOperatorName))
```

Comment:
this is bad that we manually construct the key here. This quickly diverges from the constant key used elsewhere.

Reply:
In other words: have one place where to get the constant key from.

Reply:
fixed.